### PR TITLE
change magento cli to bin/magento and add in bin info for system PATH

### PR DESCRIPTION
**Pull request by camdixon**
issue #1742 as a first pull request to make sure it's proper.  Then will update other CLI pages for bin/magento command.

whatsnew
Changed [CLI examples](http://devdocs.magento.com/guides/v2.2/config-guide/cli/config-cli-subcommands-mode.html) to include the `bin/magento` syntax and added guidance on adding `bin` to your system `PATH` to run commands from anywhere.

### CODE REVIEW

File: guides/v2.0/config-guide/cli/config-cli-subcommands-mode.md [hunk 1]
```diff
@@ -126,12 +126,17 @@ When you change from production to developer mode, you should clear generated cl
...
 
-		magento deploy:mode:set developer
+		bin/magento deploy:mode:set developer
 
 	The following message displays:
 
 		Switched to developer mode.
 
+### To run Magento commands from any directory
+<a href="{{ page.baseurl }}config-guide/cli/config-cli.html#config-install-cli-first">Run Magento system commands from anywhere</a>
```
Comment (jeff-matthews) [inline guides/v2.0/config-guide/cli/config-cli-subcommands-mode.md]: Suggest rewrite and add period at end of sentence:

> Run Magento **CLI commands** from anywhere.

Also, please use markdown syntax for links.

File: guides/v2.0/config-guide/cli/config-cli-subcommands-mode.md [hunk 2]
```diff
@@ -126,12 +126,17 @@ When you change from production to developer mode, you should clear generated cl
...
+
+If you haven't added `<magento-install-directory>/bin` to your system PATH, then you can expect an error when running the magento command by itself.
```
Comment (jeff-matthews) [inline guides/v2.0/config-guide/cli/config-cli-subcommands-mode.md]: Wrap "Path" in backticks (e.g., `PATH`) and capitalize Magento

File: guides/v2.1/config-guide/cli/config-cli-subcommands-mode.md [hunk 3]
```diff
@@ -126,12 +126,17 @@ When you change from production to developer mode, you should clear generated cl
...
 
-		magento deploy:mode:set developer
+		bin/magento deploy:mode:set developer
 
 	The following message displays:
 
 		Switched to developer mode.
 
+### To run Magento commands from any directory
+<a href="{{ page.baseurl }}config-guide/cli/config-cli.html#config-install-cli-first">Run Magento system commands from anywhere</a>
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/cli/config-cli-subcommands-mode.md]: Suggest rewrite and add period at end of sentence:

> Run Magento **CLI commands** from anywhere.

Also, please use markdown syntax for links.

File: guides/v2.2/config-guide/cli/config-cli-subcommands-mode.md [hunk 4]
```diff
@@ -126,12 +126,17 @@ When you change from production to developer mode, you should clear generated cl
...
 
-		magento deploy:mode:set developer
+		bin/magento deploy:mode:set developer
 
 	The following message displays:
 
 		Switched to developer mode.
 
+### To run Magento commands from any directory
+<a href="{{ page.baseurl }}config-guide/cli/config-cli.html#config-install-cli-first">Run Magento system commands from anywhere</a>
```
Comment (jeff-matthews) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-mode.md]: Suggest rewrite and add period at end of sentence:

> Run Magento **CLI commands** from anywhere.

Also, please use markdown syntax for links.

File: guides/v2.1/config-guide/cli/config-cli-subcommands-mode.md [hunk 5]
```diff
@@ -126,12 +126,17 @@ When you change from production to developer mode, you should clear generated cl
...
+
+If you haven't added `<magento-install-directory>/bin` to your system PATH, then you can expect an error when running the magento command by itself.
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/cli/config-cli-subcommands-mode.md]: Wrap "Path" in backticks (e.g., `PATH`) and capitalize Magento

File: guides/v2.2/config-guide/cli/config-cli-subcommands-mode.md [hunk 6]
```diff
@@ -126,12 +126,17 @@ When you change from production to developer mode, you should clear generated cl
...
+
+If you haven't added `<magento-install-directory>/bin` to your system PATH, then you can expect an error when running the magento command by itself.
```
Comment (jeff-matthews) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-mode.md]: Wrap "Path" in backticks (e.g., `PATH`) and capitalize Magento

**Comment by jeff-matthews** [comment]
No problem! All you have to do is make changes to your local branch, commit, and push. This PR will automatically update with your changes.

**Comment by camdixon** [comment]
@jeff-matthews on this page https://devdocs.magento.com/guides/v2.0/config-guide/cli/config-cli-subcommands-spt-util.html
I could not find where {% include config/code-backup.md %} (line 62) actually is to change that command.

**Comment by camdixon** [comment]
@jeff-matthews I have edited other CLI pages and your feedback requests.  Please see my previous comment for one command that I missed on the spt-util page.

**Comment by jeff-matthews** [comment]
@camdixon, the `{% include config/code-backup.md %}` file is located at `_includes/config/code-backup.md`.

File: guides/v2.1/config-guide/cli/config-cli-subcommands-depen.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
----
+bin/---
```
Comment (jeff-matthews) [inline guides/v2.1/config-guide/cli/config-cli-subcommands-depen.md]: This should be deleted.

**Comment by camdixon** [comment]
@jeff-matthews Good catch.  Can you please inspect latest changes?

**Pull request merged**